Marlin/src/lcd/menu/menu_motion.cpp

```diff
@@ -156,7 +156,8 @@ void _menu_move_distance(const AxisEnum axis, const screenFunc_t func, const int
     SUBMENU(MSG_MOVE_0001IN, []{ _goto_manual_move(IN_TO_MM(0.001f)); });
   }
   else {
-    if (LARGE_AREA_TEST) SUBMENU(MSG_MOVE_100MM, []{ _goto_manual_move(100); });
```

Comment:
`LARGE_AREA_TEST` is defined here based on >=1,000mm bed sizes: https://github.com/MarlinFirmware/Marlin/blob/5809bdd400c77ef1e25b5215ef14d07671b9aaf6/Marlin/src/lcd/menu/menu_motion.cpp#L31

If the plan is to remove the 1,000mm bed size requirement, then more updates are needed:

https://github.com/MarlinFirmware/Marlin/blob/5809bdd400c77ef1e25b5215ef14d07671b9aaf6/Marlin/src/lcd/menu/menu_motion.cpp#L84

&

https://github.com/MarlinFirmware/Marlin/blob/5809bdd400c77ef1e25b5215ef14d07671b9aaf6/Marlin/src/lcd/menu/menu_motion.cpp#L153

Reply:
Changes made and compiled on default examples. 
Updated all language files as well to add the 50mm move.
Didn't bother with the .5in as I really don't think many use anything other than metric.

Reply:
> Didn't bother with the .5in as I really don't think many use anything other than metric.

We support inch mode (`INCH_MODE_SUPPORT`), so the change should be made for both.

Reply:
Added the .5, not all language files even had inch mode defined.  So there were a lot less changes.  I am getting an error on checks on line 83 of edit on  menu_motion.cpp.  for some reason it doesn't show the error on my build, likely as it isn't used.